### PR TITLE
[Push, Wallet] Subscriptions List

### LINKED ISSUE #698: [Push] Notifications Screen
Wallet app should display active push subscriptions

### PR DESCRIPTION
**Pull request by llbartekll**
# Description
- Add Tab bar to main screen
- add Subscriptions list screen
- add active subscription publisher to push wallet client
- present session proposal and push request on top of main screen(the one with tabs)

Resolves #698 

## How Has This Been Tested?
- manually

## Due Dilligence

* [ ] Breaking change
* [ ] Requires a documentation update

### CODE REVIEW

File: Example/WalletApp/PresentationLayer/Wallet/Notifications/NotificationsRouter.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    weak var viewController: UIViewController!
+
+    private let app: Application
+
+    init(app: Application) {
+        self.app = app
+    }
+
+    func presentNotifications(subscription: WalletConnectPush.PushSubscription) {
+
```
Comment (flypaper0) [inline Example/WalletApp/PresentationLayer/Wallet/Notifications/NotificationsRouter.swift:15]: next iteration?

Comment (llbartekll) [reply Example/WalletApp/PresentationLayer/Wallet/Notifications/NotificationsRouter.swift:15]: correct

File: Sources/WalletConnectPush/Client/Wallet/PushSubscriptionsObserver.swift [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import Combine
+import Foundation
+
+class PushSubscriptionsObserver {
```
Comment (flypaper0) [inline Sources/WalletConnectPush/Client/Wallet/PushSubscriptionsObserver.swift:4]: may better to do it inside Push SDK?

**Review by flypaper0 — APPROVED** [review]

File: Sources/WalletConnectPush/Client/Wallet/PushSubscriptionsObserver.swift (hunk 2 above)
Comment (llbartekll) [reply Sources/WalletConnectPush/Client/Wallet/PushSubscriptionsObserver.swift:4]: good catch, it got to wallet files by accident 🤔

Comment (llbartekll) [reply Sources/WalletConnectPush/Client/Wallet/PushSubscriptionsObserver.swift:4]: :D it actually is in sdk already, just the wallet dir

**Review by alexander-lsvk — APPROVED** [review]

**Pull request merged**